var/spack/repos/builtin/packages/py-backports-weakref/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBackportsWeakref(PythonPackage):
+    """Backports of new features in Python's weakref module"""
+
+    homepage = "https://github.com/PiDelport/backports.weakref"
+    # Can't use pypi link as it doesn't have source files for the versions
+    # indicated below.
```

Comment:
It looks like PyPI has 1.0.post1, which is the latest version.

Reply:
I added that specific version to satisfy dependency list in a requirements file. Now I think about it is probably not a critical dependency so it should be fine to use version 1.0.post1 I guess.